docs/cli.rst

```diff
@@ -0,0 +1,45 @@
+Command Line Interface
+======================
+
+Nameko ships with a command line interface to make hosting and interacting with services as easy as possible.
+
+Running a Service
+-----------------
+
+::
+
+    $ nameko run [<module>[:<ServiceClass>]]
+
+Discover and run a service class.
```

Comment:
This isn't enough for me to know how to use Nameko in production. Which might be fine! But its definitely missing.